src/consumption.cpp

```diff
@@ -710,6 +713,33 @@ bool player::eat( item &food, bool force )
         } else {
             add_msg_player_or_npc( _( "You eat your %s." ), _( "<npcname> eats a %s." ),
                                    food.tname().c_str() );
+            if ( !spoiled && !food.has_flag( "ALLERGEN_JUNK" ) ) {
+                bool has_table_nearby = false;
+                for ( const tripoint &pt : g->m.points_in_radius( pos(), 2 ) ) {
+                    if ( g->m.has_flag_furn( "FLAT_SURF", pt ) || g->m.has_flag( "FLAT_SURF", pt ) ||
+                        ( g->m.veh_at(pt) && g->m.veh_at(pt)->vehicle().has_part( "KITCHEN" ) ) ) {
+                        has_table_nearby = true;
+                    }
+                }
+                bool eat_with_table = false;
+                if ( g->m.has_flag_furn( "CAN_SIT", pos() ) && has_table_nearby )
```

Comment:
```suggestion
                if( g->m.has_flag_furn( "CAN_SIT", pos() ) && has_table_nearby ) {
```

Please astyle your changes. Also, use curly brackets even for one-line statements.